test/client_test.dart

```diff
@@ -308,36 +308,28 @@ void main() {
       expect(eventUpdateList[5].roomID, '!726s6s6q:example.com');
       expect(eventUpdateList[5].type, EventUpdateType.timeline);
 
-      expect(eventUpdateList[6].content['type'], 'm.typing');
+      expect(eventUpdateList[6].content['type'], LatestReceiptState.eventType);
```

Comment:
can we somehow also check the content in `room.ephemerals` here?

Reply:
Good idea 👍💡